docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,42 +1,90 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
-////
-= Quarkus - Quarkus Extension for Spring DI API
 
-include::./attributes.adoc[]
-:extension-status: preview
 
-While users are encouraged to use CDI annotations for injection, Quarkus provides a compatibility layer for Spring dependency injection in the form of the `spring-di` extension.
+// comment out to show community version
+:product-docs:
+// TODO: consider moving this to the attributes file
+:doctype: book
 
-This guide explains how a Quarkus application can leverage the well known Dependency Injection annotations included in the Spring Framework.
+//Top-Level Assembly
+[id="using-the-quarkus-extension-for-spring-di-api_{context}"]
+= Using the Quarkus Extension for Spring DI API
 
-include::./status-include.adoc[]
+ifdef::context[:parent-context: {context}]
 
-== Prerequisites
+:context: using-the-quarkus-extension-for-spring-di
+
+include::./attributes.adoc[]
```

Comment:
./ seems superfluous. if it's in the same directory you don't need to explicitly set this. `include::attributes.adoc[]`

Reply:
@oraNod Thanks, Even though I left this unchanged from the upstream version, I see your point. I'll change it in the upstream attributes file.